src/c128time.c

```diff
@@ -87,3 +102,25 @@ void set_time(uint8_t day, uint8_t hour, uint8_t min, uint8_t sec) {
   /* set CIA1.tod_10 and program "Control Timer A" */
   __asm__("jsr initsystime");
 }
+
+uint8_t _daytime_irq(void) {
+  static char *t;
+  static uint8_t x, y;
+  /* We are called 60 times a second. We only want to draw a clock
+   * when we are a) on the mainscreen and b) the seconds changed */
+  if (kasse_menu == MENU_MAIN && CIA1.tod_sec != daytime.sec) {
+    t = get_time();
+    x = wherex();
+    y = wherey();
+    cputsxy(70, 3, t);
+    gotoxy(x, y);
+  }
```

Comment:
We should definitely not write to the screen from an IRQ handler. That's way too much code, IMHO.

Reply:
I don't think it is too big a deal:
- The limiting factor is not the amount of code, but the slow interface to the VDC.
- calling 1000 iterations of current code (I removed the check if seconds changed) take about 10 seconds, that's tight but still within 16ms per IRQ handler.
- directly using vdc_write_mem() takes this down to 7 seconds for 1000 interations or 7ms per IRQ handler.

There's nothing happening in the code outside the IRQ handler, it just sits there at cgetc();